src/registrar/assets/sass/_theme/_nav-primary.scss

```diff
@@ -0,0 +1,20 @@
+@use "uswds-core" as *;
```

Comment:
We decided to pull nav into it's own file so it's cleaner to see where all the nav changes are being made.